dap-python.el

```diff
@@ -205,14 +206,17 @@ overriden in individual `dap-python' launch configurations."
          (when (sequencep python-args)
            (setq python-args (mapconcat #'shell-quote-argument python-args " ")))
          (plist-put conf :program-to-start
-                    (format "%s%s -m ptvsd --wait --host %s --port %s%s %s %s"
+                    (format "%s%s -m ptvsd --wait --host %s --port %s %s"
```

Comment:
@nbfalcon not part of this review, but do we support vterm here?

Reply:
@yyoncho we could use vterm here, but not with the facilities that I have already implemented for "runInTerminal", as they depend on a debug-session. However, adding vterm support to dap-debug shouldn't be that much effort (basically `let`-bind vterm-shell to :program-to-start and then call (vterm)). I could implement it if I have some more spare time.

Reply:
It should be max. 20 lines of Elisp, probably 4-10.